doc/source/plugins/b611_django_rawsql_used.rst

```diff
@@ -0,0 +1,5 @@
+------------------------
+B610: django_rawsql_used
+------------------------
+
+.. automodule:: bandit.plugins.django_injection_sql
```

Comment:
Because you have two functions in this module, you should instead use:

.. currentmodule:: bandit.plugins.django_injection_sql

.. autofunction:: django_rawsql_used
:noindex: